general/contributing/development.md

```diff
@@ -34,47 +34,51 @@ The first step is to set up a copy of the Git repository of the project you want
 1. On GitHub, "Fork" the Jellyfin repository you wish to contribute to, to your own user account using the "Fork" button in the relevant repository.
 
 1. Clone your fork to your local machine and enter the directory:  
-    `git clone git@github.com:yourusername/projectname.git`  
-    `cd projectname/`
+   `git clone git@github.com:yourusername/projectname.git`  
+   `cd projectname/`
 
 1. Add the "upstream" remote, which allows you to pull down changes from the main project easily:  
-    `git remote add upstream git@github.com:jellyfin/projectname.git`
+   `git remote add upstream git@github.com:jellyfin/projectname.git`
 
 1. Initialize the Git submodules; most projects have at least one:  
-    `git submodule update --init`
+   `git submodule update --init`
 
     **NOTE:** Avoid editing files inside of submodules; this may result in unintended side-effects including overwriting of your work. Always modify the submodule in its own cloned repository elsewhere.
 
+1. To get the `Jellyfin.Server` project to run successfully, checkout both the main [Jellyfin project](https://github.com/jellyfin/jellyfin), as well as the [Jellyfin Web](https://github.com/jellyfin/jellyfin-web) project.
+
+1. Build the Jellyfin Web project with Yarn or NPM, and copy the location of the resulting `dist` folder. In your `Jellyfin.Server` project add an environment variable named `JELLYFIN_WEB_DIR` with the value set to the full path of your `dist` folder.
```

Comment:
You have to do either one, not both.

Reply:
Do you mind elaborating? I had to follow all of those steps to get the server running successfully from VS.

Reply:
You can either copy the ‘dist’ folder into the ‘MediaBrowser.WebDashboard’ project (don’t know the exact path rn) or just set the env variable to the path of the ‘dist’ directory.

Reply:
Ah yes, I guess I can clarify it a bit better - I meant "just set the env variable to the path of the ‘dist’ directory."

Reply:
Oh, I see now, “copy the **location**”

Reply:
Yep